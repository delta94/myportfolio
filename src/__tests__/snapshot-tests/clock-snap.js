import React from 'react';
import { shallow } from 'enzyme';
import toJSON from 'enzyme-to-json';
import Clock from '../../components/clock';

it('Render clock correctly', () => {
  const wrapper = shallow(<Clock />);
  expect(toJSON(wrapper)).toMatchSnapshot();
});
