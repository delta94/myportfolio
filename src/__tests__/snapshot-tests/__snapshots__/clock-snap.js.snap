// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render clock correctly 1`] = `
<styled.div
  className="clock-wrapper"
>
  <styled.span>
    Aug 23, 2018, 1:00 PM
  </styled.span>
</styled.div>
`;
